<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecondWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="drop_shadow_frame">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.949, x2:1, y2:0, stop:0.0995025 rgba(82, 120, 145, 255), stop:0.985075 rgba(61, 71, 91, 255));
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="Title_Bar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_title">
            <property name="font">
             <font>
              <family>Bauhaus 93</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(177, 197, 212);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_title">
               <property name="font">
                <font>
                 <family>Bahnschrift SemiLight SemiConde</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Non-Parametric Classifier</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_btns">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="btn_maximize">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>17</width>
               <height>17</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: none;
border-radius: 8px; 
	background-color: rgb(85, 255, 127);
}
QPushButton:hover {
background-color: rgba(85, 255, 127, 150);

}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_close">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>20</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>17</width>
               <height>17</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: none;
border-radius: 8px; 
	
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {
background-color: rgba(255, 0, 0, 150);

}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_minimize">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>20</y>
               <width>16</width>
               <height>16</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>17</width>
               <height>17</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: none;
border-radius: 8px; 
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {
background-color: rgba(255, 170, 0, 150);

}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Content_Bar">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>941</width>
            <height>681</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="EdgeImg_label">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>40</y>
             <width>351</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Condensed</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(104, 101, 115);
border: 2px solid rgb(177, 197, 212);
color: rgb(177, 197, 212);</string>
           </property>
           <property name="text">
            <string>Edge-Detected Image:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="InputImg_label">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>40</y>
             <width>351</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift Condensed</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(104, 101, 115);
border: 2px solid rgb(177, 197, 212);
color: rgb(177, 197, 212);</string>
           </property>
           <property name="text">
            <string>Input Image:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QFrame" name="frame_2">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>440</y>
             <width>521</width>
             <height>261</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="Prediction_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift Condensed</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(216, 185, 170);</string>
            </property>
            <property name="text">
             <string>Prediction :</string>
            </property>
           </widget>
           <widget class="QLabel" name="accuracy_label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>170</y>
              <width>161</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift Condensed</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(213, 184, 169);</string>
            </property>
            <property name="text">
             <string>Prediction Accuracy :</string>
            </property>
           </widget>
           <widget class="QLabel" name="results_label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>5</y>
              <width>181</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift Condensed</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(216, 185, 170);
border: 5px solid rgb(216, 185, 170);</string>
            </property>
            <property name="text">
             <string>Classifier Results :</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>100</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift Condensed</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(216, 185, 170);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>170</y>
              <width>171</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Bahnschrift Condensed</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(216, 185, 170);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>100</y>
             <width>381</width>
             <height>311</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
border: 2px solid rgb(177, 197, 212);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>100</y>
             <width>381</width>
             <height>311</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: none;
border: 2px solid rgb(177, 197, 212);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Credits_Bar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_label_credits">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_credits">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>0</y>
               <width>461</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift SemiLight Condensed</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(177, 197, 212);
</string>
             </property>
             <property name="text">
              <string>By : Rachelle Abdel Massih, Myriam Abou Atmeh, Sara Oud</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_grip">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
